### ISSUE TITLE
VRPN submodule build fails on debian testing

### ISSUE THREAD

**Issue by flv0** [OSVR/OSVR-Core#508]
On this debian testing system, using gcc 6.2.1, building the vrpn submodule fails with
```
[  3%] Building CXX object vendor/vrpn/CMakeFiles/vrpnserver.dir/vrpn_Connection.C.o
/home/pschmitz/development/OSVR/OSVR-Core/vendor/vrpn/vrpn_Connection.C: In function ‘int vrpn_start_server(const char*, char*, char*, const char*)’:
/home/pschmitz/development/OSVR/OSVR-Core/vendor/vrpn/vrpn_Connection.C:2540:24: error: aggregate ‘vrpn_start_server(const char*, char*, char*, const char*)::wait status’ has incomplete type and cannot be defined
             union wait status;
```
Updating the submodule to current master (9cfc1a5) makes the error go away and all of OSVR-Core build successfully. Maybe a submodule bump is required.

**Comment by rpavlik** [comment]
Yes, one is required, and there's a pull request #468 that's a bit outdated already (I updated the VRPN build system after that point).

I'll go through and make sure my CMake files are updated across the board then do the bump.

**Comment by isantop** [comment]
I'm trying to build osvr for Linux on Ubuntu 16.10 (using gcc 6.2.0), and I'm hitting the same error as above. I'm a bit new to OSVR; can someone tell me how to update the VRPN submodule to the current master version?

**Comment by flv0** [comment]
You can just go into the vendor/vrpn folder and say `git checkout master`. However if you do a `git submodule update` in the top folder afterwards, this will be reset to the pinned submodule commit (e75e92c1) so you'd have to repeat that.
